server/schedulers/evict_leader.go

```diff
@@ -193,6 +204,11 @@ func newEvictLeaderScheduler(opController *schedule.OperatorController, conf *ev
 	}
 }
 
+// EvictStores returns the IDs of the evict-stores.
+func (s *evictLeaderScheduler) EvictStoreIDs() []uint64 {
```

Comment:
why not merge this function and `getStores` into one?

Reply:
The function `EvictStoreIDs`  should be exist in `evictLeaderScheduler` since it should be public for `evictLeaderScheduler` (`Scheduler`), and the `evictLeaderSchedulerConfig` are not accessible for the package `cluster`.  
We do not implement the function in the `EvictStoreIDs` directly because all we need in this function is to visit the private attributes in the `conf`   with threadsafe.